src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/variabledeclarationusagedistance/InputVariableDeclarationUsageDistanceTryResources.java

```diff
@@ -105,6 +105,7 @@ private void method5(OutputStream out, HashMap<String, myClass> fieldStatistics)
         final boolean[] states = {fieldStatistics.isEmpty(), fieldStatistics.isEmpty()};
         Object[] keys = fieldStatistics.keySet().toArray(new String[0]);
         Object[] values = fieldStatistics.values().toArray(new myClass[0]);
+                                                              // violation above 'Distance .* is 2.'
```

Comment:
this is not expected, there is no reasonable way to move it closer.

Reply:
Hmm, I see. The problem here is this an already existing bug didn't pop up due to another bug shadowing it.

Just for this expression, there are two bugs:

1. The calculated distance is 2.
2. This is an initialization sequence.

The no. `2` bug is fixed in this PR. **This is not an initialization sequence**. Due to that bug being fixed, now this is popping up as a violation. What do you think about adding an `until..` comment here and opening a separate issue for bug no `1`?

Reply:
I am ok split this to separate issue.

Reply:
Created https://github.com/checkstyle/checkstyle/issues/13154

Reply:
I do not think we can do this to our users, it will block them to use new version, so kind of kill whole purpose of this PR (to improve this Check).
I can agree on extension of suppression and some false negatives ...... but not on false positives.